### DISCUSSION TITLE
What's a good place to discuss language questions and issues?

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#1188] [General]
Carried over from https://github.com/prql/prql/issues/1117 ...

PRQL has at least three places to discuss things:

* [Issues](https://github.com/prql/prql/issues) - a good place for bug reports, not so much for random questions.
* [Discussions](https://github.com/prql/prql/discussions) - a good place for random questions - Github ensures that they stay visible, even when they get old
* [Discord](https://discord.com/invite/eQcfaCmsNc) - a good place to get rapid answers, if a knowlegable person happens to be on-line. It's harder to track a question from a while ago

What is the sense of the group about the most effective way to use any of these? Thanks.

**Comment by max-sixty** [comment]
That seems like a good summary of the tradeoffs...

**Comment by snth** [comment]
Thank you for this question. I have been pondering the same question myself. 

When I last looked at Discussions there wasn't anything here yet, but I see we have a number of questions now. I haven't used Github Discussions before but at first glance it appears to work similarly to StackOverflow. Is that correct?

If that's the case then it seems like a good place to have more general discussions that don't pertain to a specific issue.
